src/SolutionTemplate/UnoSolutionTemplate/Skia.Gtk/UnoQuickStart.Skia.Gtk.csproj

```diff
@@ -4,6 +4,7 @@
 		<OutputType Condition="'$(Configuration)'=='Release'">WinExe</OutputType>
 		<OutputType Condition="'$(Configuration)'=='Debug'">Exe</OutputType>
 		<TargetFramework>net5.0</TargetFramework>
```

Comment:
Are we at point where we can use net6.0 in templates?

Reply:
I think it should be OK for Skia and WASM, what do you think @jeromelaban?

Reply:
Let's keep this template as-is, it's UWP and we won't make evolve much.